src/ServiceControl.AcceptanceTests/ServiceControl.AcceptanceTests.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/CodeInspection/CSharpLanguageProject/LanguageLevel/@EntryValue">CSharp71</s:String></wpf:ResourceDictionary>
```

Comment:
Are these changes really necessary? I thought we are using the latest repo dotsettings from RepoStandards on the solution level. Shouldn't the root one be enough because those settings are inherited?

Reply:
Thanks @danielmarbach, good catch. I will move this to the solution level .dotsettings file.